www-src/app/assets/backgroundservice.html

```diff
@@ -13,6 +13,6 @@
   </head>
 
   <body>
-    <script>require('service/service');</script>
+    <script>var initialize_name = 'background_service'; require('initialize');</script>
```

Comment:
@kosssi I really don't like to pollute the `window` object. At least, you should be explicit when you do so (`window.initialize_name = ...`).

As the `initialize` script as nothing in common, I think it's better to split it in two, and only require the relevant part.

If you want to want to keep only one `initialize` script, maybe you could play with [`currentScript`](https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript).